include/mbedtls/aes.h

```diff
@@ -255,30 +264,30 @@ int mbedtls_aes_crypt_ctr( mbedtls_aes_context *ctx,
 #endif /* MBEDTLS_CIPHER_MODE_CTR */
 
 /**
- * \brief           Internal AES block encryption function
- *                  (Only exposed to allow overriding it,
- *                  see MBEDTLS_AES_ENCRYPT_ALT)
+ * \brief Internal AES block encryption function. This is 
+ *        only exposed to allow overriding it,
+ *        see MBEDTLS_AES_ENCRYPT_ALT.
```

Comment:
if we are changing, I would change to:
"This is only exposed to allow overriding it, using MBEDTLS_AES_ENCRYPT_ALT."

Reply:
done